background.css

```diff
@@ -13,4 +13,15 @@ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLI
 }
 .tag-link {
   color : blue
-}
\ No newline at end of file
+}
+
+
+#humanMsg .favorited {
+
+	background: url(chrome-extension://kcdgmpophadgjfkjdefoinnbnikhlone/img/sprites.png) no-repeat scroll -324px -5px transparent;
+
+}
+
+.favorite-image.favorited, .favorite-image:active {
```

Comment:
lol i was sure there was a way to do that.  i am just clearly not that great of a web developer :)